templates/gmsim/20.4.1.4/emod3d_defaults.yaml

```diff
@@ -57,7 +59,5 @@ ts_xy: 1
 ts_xz: 0
 ts_yz: 0
 tzero: 0.6
+use_qpqs: 0
 vmodel_swapb: 0
-xseis: 0
-yseis: 0
-zseis: 0
```

Comment:
why was these originally here and now we are removing them from all gmsim template?